homeassistant/components/light/tplink.py

```diff
@@ -71,10 +64,31 @@ def __init__(self, smartbulb: 'SmartBulb', name) -> None:  # noqa: F821
         self._max_mireds = None
         self._emeter_params = {}
 
+    @property
+    def unique_id(self):
+        """Return a unique ID."""
+        return self._sysinfo["mac"]
+
     @property
     def name(self):
-        """Return the name of the Smart Bulb, if any."""
-        return self._name
+        """Return the name of the Smart Bulb."""
+        return self._sysinfo["alias"]
+
+    @property
+    def device_info(self):
+        """Return information about the device."""
+        return {
+            'identifiers': {
+                (TPLINK_DOMAIN, self.unique_id)
```

Comment:
Remove this. It's not a TP Link provided identifier, it's a Mac address and it's already provided as a connections.

Reply:
Done.